app/config/config.inc.php

```diff
@@ -37,4 +37,6 @@
     if (!defined('CHAMELEON_ENABLE_FLUSHING')) {
         define('CHAMELEON_ENABLE_FLUSHING', true);
     }
+
+    define('PKG_EXTRANET_USE_CASE_INSENSITIVE_LOGIN_NAMES', true);
```

Comment:
Checked the core code. Looks like this constant is not defined by default at all.
Constants are deprecated for configs. We should use a symfony config here later. 
As hotfix this is o.k. for me.